qrenderdoc/Windows/Dialogs/PerformanceCounterSelection.cpp

```diff
@@ -78,6 +83,7 @@ QString ToString(CounterFamily family)
     case CounterFamily::Generic: return lit("Generic");
     case CounterFamily::Intel: return lit("Intel");
     case CounterFamily::NVIDIA: return lit("NVIDIA");
+    case CounterFamily::KHR: return lit("KHR");
```

Comment:
I don't think KHR is a good public-facing name for this type of counter, since it doesn't mean anything to non-Vulkan users. I think `VulkanEXT` and `"Vulkan Extended"` would be more understandable to people not familiar with this extension.

Reply:
Done.

Reply:
To clarify the enum name needs to change to something like `VulkanEXT`, because the enum is public facing as well (e.g. via the python API).

Reply:
(See also: `IsKHRCounter`'s name and docstring, etc).

Reply:
Ah!

Reply:
Fixed.